pkg/virt-launcher/virtwrap/api/converter_test.go

```diff
@@ -1350,19 +1350,29 @@ var _ = Describe("Converter", func() {
 			Expect(domain.Spec.Devices.Interfaces[0].Model.Type).To(Equal("e1000"))
 		})
 
-		It("should set nic pci address when specified", func() {
+		table.DescribeTable("should set nic pci address when specified", func(sriov *v1.InterfaceSRIOV) {
 			v1.SetObjectDefaults_VirtualMachineInstance(vmi)
 			vmi.Spec.Domain.Devices.Interfaces[0].PciAddress = "0000:81:01.0"
-			test_address := Address{
+			expectedPCIAddress := Address{
 				Type:     "pci",
 				Domain:   "0x0000",
 				Bus:      "0x81",
 				Slot:     "0x01",
 				Function: "0x0",
 			}
+			iface := vmi.Spec.Domain.Devices.Interfaces[0]
```

Comment:
```suggestion
			iface := &vmi.Spec.Domain.Devices.Interfaces[0]
```

Otherwise, the introduced code is not executed from the test.

Reply:
I do not understand why we need to use the a pointer to the iface instead of the iface itself. Could you please clarify?

Reply:
If we don't use a pointer here the change to `iface` is not propagated. I realized this when running the test locally, expecting the code branch introduced would be covered, but it wasn't.

Reply:
Ohh... I think I understand what you mean. This is really bad!!

Reply:
Done